.github/workflows/ci.yml

```diff
@@ -1083,7 +1083,10 @@ jobs:
           ln -s -f "$HOME/.tap/tap" /usr/local/bin/tap
       - name: Debug
         run: ls -la ./publish
-      - name: Install PackagePublish
-        run: tap package install -f PackagePublish --version beta
-      - name: Publish
-        run: tap package publish -r http://packages.opentap.io -k ${{ secrets.REPO_PASS }} ./publish/*.TapPackage
+      - name: Setup OpenTAP
+        uses: opentap/setup-opentap@main
+        with:
+          version: 9.25.1
+          packages: 'Repository Client:1.0.3-beta.2'
+      - name: Publish Packages
+        run: tap repo upload --repository https://packages.opentap.io --token ${{ secrets.PUBLIC_REPO_TOKEN }} /publish/*.TapPackage
```

Comment:
`PUBLIC_REPO_TOKEN` is not configured in this repository. Can you revert to `REPO_PASS`?